lm75bd/lm75bd.c

```diff
@@ -26,7 +26,28 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+
+  if (temp == NULL) {
+    return ERR_CODE_INVALID_ARG; 
+  }
+  
+  uint8_t temp1 = TEMP_ADDR;
+  i2cSendTo(devAddr, &temp1, 1);
```

Comment:
make sure to check if function returns a success or not. Feel free to use the RETURN_IF_ERROR_CODE macro in logging.h.